web/src/locales/es.json

```
@@ -80,9 +80,8 @@
   "estimation-card": {
     "link": "Más información sobre nuestros modelos de estimación",
     "estimated_time_slicer_average": {
-      "title": "Los datos son estimados",
-      "pill": "Retraso",
-      "body": "Los datos de esta hora aún no se han transmitido. Los valores mostrados son estimaciones y se reemplazarán por mediciones cuando estén disponibles."
+      "title": "Los datos son preliminares",
+      "body": ""
```

We need some help updating the body text here from a Spanish speaker :)

---

hey @madsnedergaard, 
The "title" seems fine to me :) 
Regarding the "body", do we still want to have the "body" as it was before? I validated it some time ago with @silkeholmebonnen, so we can take that one again.

---

Brilliant, thanks! 🙏 
```suggestion
      "body": ""Los datos para esta hora aún no se han transmitido. Los valores mostrados son estimaciones calculadas a partir de valores históricos y se reemplazarán cuando los datos reales estén disponibles. Es posible que los valores estimados no tengan en consideración fenómenos meteorológicos en tiempo real."
```